nucypher/cli/actions.py

```diff
@@ -606,3 +607,26 @@ def get_or_update_configuration(emitter, config_class, filepath: str, config_opt
             emitter.message(f"Updated configuration values: {', '.join(updates)}", color='yellow')
             config.update(**updates)
         return emitter.echo(config.serialize())
+
+
+def select_worker_config_file(emitter, config_file, worker_address, provider_uri, network, federated):
+
+    config_root = abspath(dirname(config_file)) if config_file else DEFAULT_CONFIG_ROOT
+    worker_config_exists = glob.glob(UrsulaConfiguration.default_filepath(config_root=config_root))
+
+    if not worker_config_exists:
+        emitter.message("No Ursula configurations found.  run 'nucypher ursula init' then try again.", color='red')
+        raise click.Abort()
+
+    # TODO: Needs Cleanup
+    more_than_one_worker_config_exists = glob.glob(f'{config_root}/ursula-0x*.json')
+    ethereum_account_required = not worker_address and not federated
+
+    if more_than_one_worker_config_exists:
+        if config_file is None:
+            if ethereum_account_required:
+                worker_address = select_client_account(emitter=emitter, network=network, provider_uri=provider_uri)
```

Comment:
Should we print something about the user having multiple configuration accounts, and therefore we need them to specify the correct worker, before just prompting them for the worker?

Reply:
:heavy_check_mark: